test/support/helpers/water/licence-version.helper.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+/**
+ * @module LicenceVersionHelper
+ */
+
+const { generateUUID } = require('../../../../app/lib/general.lib.js')
+const LicenceVersionModel = require('../../../../app/models/water/licence-version.model.js')
+
+/**
+ * Add a new licence version
+ *
+ * If no `data` is provided, default values will be used. These are
+ *
+ * - `licenceId` - [random UUID]
+ * - `issue` - 1
+ * - `increment` - 0
+ * - `status` - 'current'
+ * - `startDate` - new Date('2022-01-01')
+ * - `externalId` - '9:99999:1:0'
+ * - `createdAt` - new Date()
+ * - `updatedAt` - new Date()
+ *
+ * @param {Object} [data] Any data you want to use instead of the defaults used here or in the database
+ *
+ * @returns {module:LicenceVersionModel} The instance of the newly created record
+ */
+async function add (data = {}) {
+  const insertData = defaults(data)
+
+  return LicenceVersionModel.query()
+    .insert({ ...insertData })
+    .returning('*')
+}
+
+/**
+ * Returns the defaults used when creating a new licence version
```

Comment:
https://github.com/DEFRA/water-abstraction-system/commit/1f972f33f9accb029de6393d4fcf177ca98ce772#diff-327dd98523aaf70b0b601aa777be715df63cb4922a9061e29006f4512ce60c3d

I think in one of Alan's recent PR's we are removing anything after 'Returns the defaults used'

Reply:
Alan made the default data more random to help with the Jest switch. He hasn't removed it though from what I can tell. Is that right @Cruikshanks ?

Reply:
Sorry not the actual data just the comment on line 37 'Returns the defaults used when creating a new licence version'

Reply:
Nope,@Beckyrose200 is right. I used the fact I needed to touch most helpers to make the comment generic and remove the possibility of copy-and-paste errors creeping in.

Reply:
Your comment has ended up with me spotting an issue with the helper though. We have a timestamp method Alan created to generate the `createdAt` and `updatedAt` that I'm not using. I'll add that

Reply:
Cheers @Beckyrose200 I've sorted it out now. Thought you were saying to remove the defaults 😀